wcomp/foxes_interface.py

```diff
@@ -319,13 +324,22 @@ def read_analyses(self, analyses, mbook, farm, states, keymap={}, **algo_pars):
         yaw_angles = np.array([analyses["yaw_angles"]])
 
         wes_analysis = analyses
-        _velocity_model_mapping = WAKE_MODEL_MAPPING[wes_analysis["wake_model"]["velocity"]["name"]]
+        wake_model_name = wes_analysis["wake_model"]["velocity"]["name"]
+        _velocity_model_mapping = WAKE_MODEL_MAPPING[wake_model_name]
         _velocity_model = _velocity_model_mapping["model_ref"]
         _velocity_model_parameters = {
-            k: wes_analysis["wake_model"]["velocity"]["parameters"][v]
+            k: wes_analysis["wake_model"]["velocity"]["parameters"].get(v, v)
```

Comment:
What's the reason for using the default "v" here? This chunk of code is meant to map the parameters from the [windIO file](https://github.com/rafmudaf/wcomp/blob/main/examples/cases_torque2024/bastankhah2016/wind_energy_system.yaml#L11) to their corresponding name in the software that are integrated to wcomp. The mapping is set up in the [WAKE_MODEL_MAPPING](https://github.com/rafmudaf/wcomp/blob/main/wcomp/foxes_interface.py#L38) parameter.

I saw you added `axial_induction` as a parameter to the Bastankhah models for foxes, but it's not a parameter in the windIO input files. The change in this line looks like it supports adding `axial_induction: Betz` into the parameter list. Is that correct?

Reply:
Yes, the induction is "Madsen" by default - should not make a big difference, but I think this may be different in floris (and it is a deviation from the paper). You are right, this has nothing to do with windio - feel free to deal with it by any other means.

Concerning the `v` parameter default, this was necessary during debugging when I set "kb" to the numerical value instead of the string in the WAKE_MODEl_MAPPING. You can change that back, sorry - all of this is a bit hasty

Reply:
Thanks @SchmJo I've made these changes